code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -260,10 +260,12 @@
 	var/mob/living/carbon/target_collateral_carbon
 	var/obj/structure/table/target_table
 	var/obj/machinery/disposal/bin/target_disposal_bin
+	var/obj/structure/closet/target_storage_container
```

Comment:
This is sort of getting ridiculous, make this signal based (and for brownie points, make everything else signals too).